package.json

```diff
@@ -17,13 +17,13 @@
   },
   "lint-staged": {
     "**/*.js": [
-      "yarn test",
+      "yarn test --passWithNoTests",
```

Comment:
ich vermute, das `--passWitNoTests` sorgt dafür, dass man nun committen kann, ohne dass die tests durchlaufen. wieso dann überhaupt noch `yarn test` und nicht einfach die zeile rauslöschen?

Reply:
Er lässt nur die test durchlaufen die berührt wurden und sich deshalb in den staged files befinden. Das heißt wir sollten eh am Besten nochmal die tests lokal durchlaufen lassen. `--passWithNoTests` ist dafür dass es nicht knallt falls sich keine Testfiles in den gestaged files befineden

Reply:
Wenn die tests noch nicht ready sind benutze:
```
describe.skip('test', () => {
});
```